packages/eslint-plugin/configs/recommended.js

```diff
@@ -1,3 +1,40 @@
 module.exports = {
-	extends: [ require.resolve( './recommended-with-formatting.js' ), 'eslint-config-prettier' ],
+	parser: 'babel-eslint',
+	extends: [
+		require.resolve( './jsx-a11y.js' ),
+		require.resolve( './custom.js' ),
+		require.resolve( './react.js' ),
+		require.resolve( './esnext.js' ),
+		'eslint-config-prettier',
```

Comment:
This single line is the only change to the original ESLint config required.

The `recommended-with-formatting.js` is not required and has been deleted

cc @jsnajdr